### ISSUE TITLE
Poker Stars Support

### ISSUE THREAD

**Issue by frrrizzz** [dickreuter/Poker#60]
Hi,

Thank you for sharing this project.
Is Poker Stars already supported? The project title says so, while in the wiki the statement is more like „nearly done“.

Kind regards

**Comment by dickreuter** [comment]
Yes it’s supported. Also try out the branch with the additional table scraping.

**Comment by frrrizzz** [comment]
What are the differences between the branches?

**Comment by siaro** [comment]
Mine doesnt recognize pokerstars. Also performs no actions on partypoker

**Comment by siaro** [comment]
The mouse just moves randomly but nothing is clicked.

**Comment by dickreuter** [comment]
What branch are you using? The current master? I reverted some changes two days ago.

**Comment by siaro** [comment]
Yes I got it to work now thank you. Is this bot using deepmind AI? Or is it just applying a preset algorithm

**Comment by dickreuter** [comment]
New version now available
